Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue

```diff
@@ -512,19 +564,45 @@ export default class SidebarComponent extends Vue {
 
     private toggleOpen() {
         this.toggleSidebar();
+        this.hideShowPopoverOnAddANoteRow();
     }
 
-    private loadName(): void {
+    private loadUserProfile(): void {
         this.authenticationService.getOidcUserProfile().then((oidcUser) => {
             if (oidcUser) {
                 this.name = this.getFullname(
                     oidcUser.given_name,
                     oidcUser.family_name
                 );
+                this.hdid = oidcUser.hdid;
+                this.loadUserPreference();
             }
         });
     }
 
+    private hideShowPopoverOnAddANoteRow(): void {
+        if (this.dismissedMyNotePopover === true) {
+            (this.$refs.popover as Vue).$emit("close");
+        } else {
+            (this.$refs.popover as Vue).$emit("open");
+        }
+    }
+
+    private loadUserPreference(): void {
+        console.log(
+            "Loading user preference (including dismissedMyNotePopover)..."
```

Comment:
nit "Loading user preference" is sufficient. Or not log it at all since it will go to prod with this